Configuration/AlCa/python/autoCond.py

```diff
@@ -79,7 +79,7 @@
     # GlobalTag for MC production with realistic conditions for Phase1 2024
     'phase1_2024_realistic'    : '111X_mcRun3_2024_realistic_v7', # GT containing realistic conditions for Phase1 2024
     # GlobalTag for MC production with realistic conditions for Phase2
-    'phase2_realistic'         : '111X_mcRun4_realistic_T15_v4'
+    'phase2_realistic'         : '111X_mcRun4_realistic_Candidate_2021_02_11_16_39_55'
```

Comment:
Hi @Sam-Harper, I created the versioned GT, which is equivalent to your candidate:

https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/111X_mcRun4_realistic_Candidate_2021_02_11_16_39_55/111X_mcRun4_realistic_T15_v5

Please update autoCond with the new GT.

Reply:
thank you!